debs/Debian/amd64/Dockerfile

```diff
@@ -13,9 +13,9 @@ RUN echo "deb http://archive.debian.org/debian/ wheezy contrib main non-free" >
     libdb5.1=5.1.29-5 libdb5.1-dev libssl1.0.0=1.0.1e-2+deb7u20 procps gawk libsigsegv2
 
 # Add Debian's source repository and, Install NodeJS 6
-RUN apt-get update && apt-get build-dep python3.2 -y && \
+RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - && \
```

Comment:
Why add nodesource GPG key?

Reply:
These changes are necessary so the Docker container can be built.